server/controllers/configurationsController/index.js

```diff
@@ -25,11 +22,8 @@ const ConfigurationsController = {
       const { config_id } = req.params
       const { deletedCount } = await ConfigModel.destroy(appDb, config_id)
 
-      if (deletedCount > 0) {
-        return res.status(200)
-      } else {
-        return res.status(404)
-      }
+      if (deletedCount >= 1) return res.status(200).end()
+      else return res.status(404).end()
```

Comment:
you could use a ternary here.

```suggestion
      return deletedCount >= 1 ? res.status(200).end() : res.status(404).end()
```

Reply:
[d31c716](https://github.com/AMP-SCZ/dpdash/pull/310/commits/d31c716b584e340b90068954fc940c49761e52fe)